test/views/DeclarationInformationViewSpec.scala

```diff
@@ -67,11 +68,13 @@ class DeclarationInformationViewSpec extends UnitViewSpec with Injector {
 
   private val submission: Submission = submission()
 
-  private val zone: ZoneId = ZoneId.of("Europe/London")
+  private val dateTimeIssued = ZonedDateTime.now
+  private val dateTimeAsText = ViewDates.formatDateAtTime(dateTimeIssued)
```

Comment:
Using these values in the test instead of the old literal values seems to invalidate the usefulness of these test as you are using the same function `ViewDates.formatDateAtTime` to create the expected result string as the view that is being tested.

Reply:
`formatDateAtTime` is tested in `test/views/helpers/ViewDatesSpec.scala`.
The tests here are mainly to verify that the date is shown, nor really to verify their format.